frontend/CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fixed combobox loader not working in muscleGroup selector
 - Fixed issue where weigh ins page showed an error if user had no weigh ins.
 - Fixed issue where you could not leave maxReps as 0.
+- Fixed Loader bug when selecting size medium
```

Comment:
Be more specific with the description. Not a megillah but like lets the bug was that it crashed the app.

Fixed issue where the app would crash if a medium sized Loader was chosen.

A bug like this is very case specific and would only be noticed by a developer during development so i wouldn't put it into the changelog. Changelog is more for bugs that the user is facing.